runelite-api/src/main/java/net/runelite/api/widgets/WidgetInfo.java

```diff
@@ -506,6 +506,10 @@
 
 	GENERIC_SCROLL_TEXT(WidgetID.GENERIC_SCROLL_GROUP_ID, WidgetID.GenericScroll.TEXT),
 
+	AGI_LAPS_CONTAINER(WidgetID.GENERIC_SCROLL_GROUP_ID, WidgetID.LapsContainer.CONTAINER),
+	AGI_LAPS_HEADER(WidgetID.GENERIC_SCROLL_GROUP_ID, WidgetID.LapsContainer.HEADER),
+	AGI_LAPS_COUNT(WidgetID.GENERIC_SCROLL_GROUP_ID, WidgetID.LapsContainer.LAPCOUNT),
```

Comment:
Given the script which generates these widgets appears to be generic (and not specific to this agility laps interface/text), these names are almost certainly wrong.